src/test/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTOTest.java

```diff
@@ -0,0 +1,41 @@
+package edu.cornell.kfs.concur.eventnotification.rest.xmlObjects;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+
+import javax.xml.bind.JAXBException;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.cornell.kfs.sys.service.CUMarshalService;
+import edu.cornell.kfs.sys.service.impl.CUMarshalServiceImpl;
+
+public class ConcurEventNotificationDTOTest {
+    private static final String EXAMPLE_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/xmlObjects/fixture/event-notification-payload-example.xml";
+    private static final String EQUAL_ASSERT_STATEMENT = "Expected value should equal the actual value.";
```

Comment:
I do not think this is used? Do we need to keep it?

Reply:
removed